AppKit/CPCompatibility.j

```diff
@@ -30,7 +30,8 @@ CPGeckoBrowserEngine                    = 1;
 CPInternetExplorerBrowserEngine         = 2;
 CPKHTMLBrowserEngine                    = 3;
 CPOperaBrowserEngine                    = 4;
-CPWebKitBrowserEngine                   = 5;
+CPWebKitBrowserEngine                   = 5;  // Safari
+CPBlinkBrowserEngine                    = 6;  // Chrome
```

Comment:
We need to be careful because this means that the engine checks against  CPWebKitBrowserEngine will now fail with recent versions of Chrome. I don't think this is what we want.